scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixEvaluation.java

```diff
@@ -4,19 +4,31 @@
 
 public interface ScalafixEvaluation {
     /**
-     * @return boolean true if scalafix has successfully been evaluated on all files configured and no ScalafixError has been found
+     * @return boolean true if scalafix has successfully been evaluated on all files configured with no error.
```

Comment:
From what I see in the implementation, this won't be set to error if we have a stale semanticdb for a file, is that correct?
```suggestion
     * @return boolean true if scalafix ran on at least one file; there might be errors in file evaluation though
```